themes/scss/infowindow/cartodb-infowindow-default.scss

```diff
@@ -5,312 +5,299 @@
+  background: #888 !important;
+}
+
+.is-header .CDB-infowindow-mediaTitle {
   bottom: 12px;
 }
 
-.Cdb-infowindow-media-title {
+.CDB-infowindow-mediaTitle {
   position: absolute;
   right: 24px;
   bottom: 24px;
   left: 24px;
 }
-
-.Cdb-infowindow-header-media {
+.CDB-infowindow-headerMedia {
   position: relative;
   overflow: hidden;
   z-index: 1;
-}
-
-.Cdb-infowindow-media-item {
-  display: block;
-  width: 100%;
-}
-
-
-.Cdb-infowindow-header-media {
-
-  .Cdb-infowindow-title > span {
+  .CDB-infowindow-title > span {
     display: inline;
     padding: 2px 0;
-    background-color: rgba(0, 0, 0, .8);
-    box-shadow: 4px 0 0 rgba(0, 0, 0, .8), -4px 0 0 rgba(0, 0, 0, .8);
+    background-color: rgba(0, 0, 0, 0.8);
+    box-shadow: 4px 0 0 rgba(0, 0, 0, 0.8), -4px 0 0 rgba(0, 0, 0, 0.8);
   }
-  .Cdb-infowindow-subtitle {
+  .CDB-infowindow-subtitle {
     margin-bottom: 4px;
   }
-  .Cdb-infowindow-subtitle > span {
+  .CDB-infowindow-subtitle > span {
     display: inline;
     padding: 2px 0;
-    background-color: rgba(0, 0, 0, .64);
-    box-shadow: 4px 0 0 rgba(0, 0, 0, .64), -4px 0 0 rgba(0, 0, 0, .64);
+    background-color: rgba(0, 0, 0, 0.64);
+    box-shadow: 4px 0 0 rgba(0, 0, 0, 0.64), -4px 0 0 rgba(0, 0, 0, 0.64);
   }
 }
-
-.Cdb-hook {
+.CDB-infowindow-media-item {
+  display: block;
+  width: 100%;
+}
+.CDB-hook {
   position: absolute;
-  left: 24px;
   bottom: 1px;
+  left: 24px;
   z-index: 10;
 }
-
-.Cdb-hook:before {
-  content:"";
+.CDB-hook:before {
   position: absolute;
-  left: 0;
   top: 0;
+  left: 0;
   width: 0;
   height: 0;
   border-right: 24px solid transparent;
+  content: '';
   z-index: 3;
 }
-
-.Cdb-hook--green.Cdb-hook:before{
+.CDB-hook--green.CDB-hook:before {
   border-top: 16px solid #98E0A8 !important;
 }
-
-.Cdb-hook--orange.Cdb-hook:before{
+.CDB-hook--orange.CDB-hook:before {
   border-top: 16px solid #E68165 !important;
 }
-
-.Cdb-hook:after, .Cdb-hook-image:after {
-  content:"";
+.CDB-hook:after,
+.CDB-hook-image:after {
   position: absolute;
-  left: 0;
   top: 3px;
+  left: 0;
   width: 0;
   height: 0;
-  border-top: 16px solid rgba(0, 0, 0, .14);
+  border-top: 16px solid rgba(0, 0, 0, 0.14);
   border-right: 24px solid transparent;
   -webkit-filter: blur(2px);
   filter: blur(2px);
+  content: '';
   z-index: 2;
 }
-
-.is-header-image {
-
-  .Cdb-hook-image {
-    position: absolute;
-    height: 16px;
-    width: 24px;
-    z-index: 3;
-    left: 24px;
-
-    img {
-      margin-left: -24px;
-    }
-  }
-  .Cdb-hook-image-inner {
-    width: 100%;
-    height: 100%;
-    overflow: hidden;
-    position: relative;
-    z-index: 3;
+.CDB-infowindow.is-header-image .CDB-hook-image {
+  position: absolute;
+  left: 24px;
+  width: 24px;
+  height: 16px;
+  z-index: 3;
+  img {
+    margin-left: -24px;
   }
 }
 
-.Cdb-infowindow-tabs {
+.CDB-infowindow.is-header-image .CDB-hook-image-inner {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
```

Comment:
Properties should be ordered position, width, height, background, overflow, z-index